core/capabilities/ccip/ocrimpls/contract_transmitter.go

```diff
@@ -23,54 +23,63 @@ type ToCalldataFunc func(
 	report ocr3types.ReportWithInfo[[]byte],
 	rs, ss [][32]byte,
 	vs [32]byte,
-) (any, error)
+) (string, string, any, error)
+
+func NewToCommitCalldata(defaultMethod, priceOnlyMethod string) ToCalldataFunc {
```

Comment:
Godoc on this would be good to have, to explain its existence vs. lack of one for exec.

Reply:
Also `NewToCommitCalldataFunc` seems to be a better name given the ret val